Sources/MultiNodeTestKit/MultiNodeTestKit+Control.swift

```diff
@@ -75,6 +93,7 @@ extension MultiNodeTest {
 // MARK: Run pieces of code on specific node
 
 extension MultiNodeTest.Control {
+    /// Runs a piece of code only on the specified `node`.
```

Comment:
🤔 the comment doesn't seem to fit the code?

Reply:
Oh right needs a small rewording, on it

Reply:
```suggestion
    /// Returns `true` if the multi-node execution is currently running on the same
    /// `node` as is passed to this function. This allows for executing pieces of code, 
    /// only on a specific node in the multi-node test execution.
```